evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -69,7 +83,8 @@ def add_result(self, score_to_minimize, pipeline):
             score_to_minimize (float): The score obtained by this pipeline on the primary objective, converted so that lower values indicate better pipelines.
             pipeline (PipelineBase): The trained pipeline object which was used to compute the score.
         """
-        super().add_result(score_to_minimize, pipeline)
+        if pipeline.model_family != ModelFamily.ENSEMBLE:
+            super().add_result(score_to_minimize, pipeline)
```

Comment:
We don't want to add results to tuner for stacked ensemble

Reply:
Got it 👍  I think its totally fine if we don't add tuners for ensemblers for the time being.

Long-term, it would be great to treat ensembles just like any other pipeline, meaning they have a tuner. For the stacked ensembler, I suppose we could search the final estimator, and the parameters of that estimator (not currently exposed, which is fine). I wonder if there are other algorithm parameters we'll want to expose.